rest-api/alembic/versions/c0e5a9912dca_add_1sal2_code.py

```diff
@@ -0,0 +1,70 @@
+"""add 1sal2 code
+
+Revision ID: c0e5a9912dca
+Revises: 2be6f6d054e8
+Create Date: 2018-04-04 13:42:58.256740
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import model.utils
+from sqlalchemy.dialects import mysql
+
+from participant_enums import PhysicalMeasurementsStatus, QuestionnaireStatus, OrderStatus
+from participant_enums import WithdrawalStatus, SuspensionStatus
+from participant_enums import EnrollmentStatus, Race, SampleStatus, OrganizationType
+from participant_enums import MetricSetType, MetricsKey
+from model.site_enums import SiteStatus, EnrollingStatus
+from model.code import CodeType
+
+# revision identifiers, used by Alembic.
+revision = 'c0e5a9912dca'
+down_revision = '2be6f6d054e8'
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+def upgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('participant_summary', sa.Column('sample_order_status_1sal2', model.utils.Enum(OrderStatus), nullable=True))
+    op.add_column('participant_summary', sa.Column('sample_order_status_1sal2_time', model.utils.UTCDateTime(), nullable=True))
+    op.add_column('participant_summary', sa.Column('sample_status_1sal2', model.utils.Enum(SampleStatus), nullable=True))
+    op.add_column('participant_summary', sa.Column('sample_status_1sal2_time', model.utils.UTCDateTime(), nullable=True))
+    op.alter_column('participant_summary', 'last_modified',
```

Comment:
Again, what's this doing in this PR?

Seems like we have a problem where we're always generating this migration...